src/Uno.UI/UI/LayoutHelper.cs

```diff
@@ -38,36 +38,50 @@ internal static void Deconstruct(this Size size, out double width, out double he
 		[Pure]
 		internal static (Size min, Size max) GetMinMax(this IFrameworkElement e)
```

Comment:
NOTE: Consider the following situation:

- e.Height: NaN
- e.MinHeight: 0
- e.MaxHeight: 100

Previously, the minSize.Height would be zero.
Now, it will be `Max(Min(maxHeight, height), minHeight)` = `Max(Min(100, inf), 0)` = `Max(100, 0)` = 100

So this is a behavior change that matches WinUI. The visible effect is not yet known. Waiting for screenshot comparison results from CI.

Reply:
Nothing is interesting in screenshot comparison results. But I think it's a good idea to merge since this matches WinUI.